pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/MailboxService.java

```diff
@@ -45,14 +37,9 @@
 public class MailboxService {
   private static final Logger LOGGER = LoggerFactory.getLogger(MailboxService.class);
   private static final int DANGLING_RECEIVING_MAILBOX_EXPIRY_SECONDS = 300;
-  private static final int DANGLING_EXCHANGE_EXPIRY_SECONDS = 300;
 
-  /**
-   * Cached receiving mailboxes that contains the received blocks queue.
-   *
-   * We use a cache to ensure the receiving mailbox are not leaked in the cases where the corresponding OpChain is
-   * either never registered or died before the sender finished sending data.
-   */
+  // We use a cache to ensure the receiving mailbox are not leaked in the cases where the corresponding OpChain is
+  // either never registered or died before the sender finished sending data.
```

Comment:
nit: Why do you use normal comments instead of javadoc here?

Reply:
I thought javadoc only applies to class and method, but seems that is not the case. Let me change it back